### DISCUSSION TITLE
Text editor cursor not in right position

### DISCUSSION THREAD

**Discussion by veronicavannini** [clientIO/joint#2762] [Q&A] [answered]
### Introduction

Hi!
I was using the TextEditor and it worked just fine.
I wanted to add a max width to the shape using 
```
node.attr("label/textWrap", {
    width: 200,
    height: null,
});
```
But after that the cursor seams misaligned from where I am writing. When I press space multiple times the cursor goes forward and doesn't write spaces, and when I write, the letters are in the point where I was before pressing space.

To reproduce the problem I just added this attribute
```
 textWrap: {
      width: 200,
      height: 400
  }

``` 
to line 40 of this demo file: `apps/RichTextEditor/ts/src/shapes.ts`

https://github.com/user-attachments/assets/5b6a0015-8b84-4ff3-95d3-5400b58a6c0e

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
Please see https://github.com/clientIO/joint/issues/2067#issuecomment-1460696444.
